src/handle_config.cpp

```diff
@@ -290,10 +316,11 @@ void handle_config(Ucfb_reader config, File_saver& file_saver, std::string_view
 {
    const auto name_hash =
       config.read_child_strict<"NAME"_mn>().read_trivial<std::uint32_t>();
+   auto actualFileName = lookup_fnv_hash(name_hash);
```

Comment:
Same style issue as before. This just needs to be made snake case or even just made `name` since the function is so small it probably won't be ambiguous. But there's also no wrong move in making it longer and more clear.
```suggestion
   const auto name = lookup_fnv_hash(name_hash);
```